internal/controller/numaresourcesscheduler_controller.go

```diff
@@ -293,6 +310,22 @@ func (r *NUMAResourcesSchedulerReconciler) syncNUMASchedulerResources(ctx contex
 	return schedStatus, nil
 }
 
+func preNormalize(spec *nropv1.NUMAResourcesSchedulerSpec, version platform.Version) {
+	parsedVersion, _ := platform.ParseVersion(activePodsResourcesSupportSince)
+	ok, err := version.AtLeast(parsedVersion)
+	if err != nil {
+		klog.Infof("failed to compare version %v with %v, err %v", parsedVersion, version, err)
+	}
+
+	if !ok {
+		return
+	}
+
+	if spec.SchedulerInformer == nil {
+		spec.SchedulerInformer = ptr.To(nropv1.SchedulerInformerShared)
+		klog.InfoS("SchedulerInformer default is overridden", "PlatformVersion", version.String(), "kubeletPodResourcesListACtivePodsByDefault", "enabled", "SchedulerInformer", &spec.SchedulerInformer)
```

Comment:
yes, we should log but probably at levels >= 4 and probably `kubeletPodResourcesListACtivePodsByDefault` should not be mentioned

Reply:
good point, thanks, fixed